scripts/compress_image.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+if [ "$#" -ne 1 ]; then
+  echo "Usage: $0 <input_image.png>"
+  exit 1
+fi
+
+input_file="$1"
+backup_file="${input_file%.png}.bak.png"
+output_file="$input_file"
+
+cp "$input_file" "$backup_file"
+
+magick "$input_file" -quality 100 -define png:compression-level=9 -strip -colors 256 "$output_file"
```

Comment:
hmmm 

```
[nix-shell:~/code/experiments/status-mobile]$ magick
bash: magick: command not found
```

we can add `magic` to default nix shell so that its available.

Reply:
thank you, added